src/plugins/vis_type_table/public/components/table_vis_cell.tsx

```diff
@@ -23,6 +23,17 @@ export const getTableVisCellValue = (
   const rawContent = row[columnId];
   const colIndex = columns.findIndex((col) => col.id === columnId);
   const htmlContent = columns[colIndex].formatter.convert(rawContent, 'html');
+  dompurify.addHook('uponSanitizeAttribute', (node, event) => {
+    if (event.attrName === 'target') {
+      event.forceKeepAttr = true;
+    }
+  });
+
+  dompurify.addHook('afterSanitizeElements', (node) => {
+    if (node instanceof Element && node.tagName?.toUpperCase() === 'A') {
+      node.setAttribute('rel', 'noopener noreferrer');
```

Comment:
will this always open stuff into a new tab.

 i think this feature is at the index pattern field level and it is configurable. does this respect this setting?

Reply:
yes, it still respects the settings at index pattern field level.  @kavilla 

For example, i set one field with `open in new tab` turned on, and another field with `open in new tab` turned off. 

https://github.com/user-attachments/assets/fefe5b0f-51a3-441f-86fa-5790a20f9064

Reply:
While `noopener` will not have an impact on anchors without a target that opens in a new context, having `noreferrer` for all links will inhibit proper logging of internal accesses; all requests will be logged without a referrer.

We should check the `target` attribute of `A` and only add `rel="noopener noreferrer"` if a `target` is missing or is not `_self`.